OHM-Trade-Agent-v1/app/services/qualified_alert_outbox.py

```
@@ -0,0 +1,402 @@
+            ),
+            journey_id=payload.get("journey_id"),
+            signal_id=payload.get("signal_id"),
+            trade_id=trade_id,
+        )
+    except Exception:
+        # Audit transport/storage must not keep a corrupt outbox row alive.
+        pass
+    try:
+        terminalize_pending_setup(trade_id, "delivery_malformed")
+    except Exception:
+        pass
```

Ruff (0.16.2)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)